extensions/wikia/ExactTargetUpdates/tasks/ExactTargetUserUpdateDriver.php

```diff
@@ -0,0 +1,59 @@
+<?php
+namespace Wikia\ExactTarget;
+
+class ExactTargetUserUpdateDriver {
+
+	/**
+	 * Allow to create when new email exists and is different than old one
+	 * @param string $oldEmail Current email from ExactTarget database
+	 * @param string $newEmail New email from Wikia to be updated
+	 * @return bool
+	 */
+	public static function shouldCreateAsEmailChanged( $oldEmail, $newEmail ) {
```

Comment:
I think we'd loose clearance of already bit complex condition logic here: https://github.com/Wikia/app/blob/SUS-123/extensions/wikia/ExactTargetUpdates/tasks/ExactTargetUserTask.class.php#L37